spyder/plugins/ipythonconsole/scripts/conda-activate.bat

```diff
@@ -11,10 +11,10 @@ set SPYDER_KERNEL_SPEC=%4
 chcp 65001>nul
 
 :: Activate kernel environment
-echo %CONDA_ACTIVATE_SCRIPT%| findstr /e "micromamba.exe">Nul && goto micromamba || goto conda
+echo %CONDA_ACTIVATE_SCRIPT%| findstr /e "conda.exe">Nul && goto standalone || goto conda
 
-:micromamba Activate using micromamba
-for /f %%i in ('%CONDA_ACTIVATE_SCRIPT% shell activate %CONDA_ENV_PATH%') do set SCRIPT=%%i
+:standalone Activate using standalone conda executable
+for /f %%i in ('%CONDA_ACTIVATE_SCRIPT% shell.cmd.exe activate %CONDA_ENV_PATH%') do set SCRIPT=%%i
```

Comment:
Yes. Although these changes may be moot after the discussion regarding `conda.exe`. In this PR, I've since changed Spyder's conda executable to the included `mamba`, so the `:standalone` block will never be executed.